input/pagecontent/authentication.md

```diff
@@ -0,0 +1,279 @@
+### Introduction
+
+Authentication is the process of verifying the identity of an entity. In the context of healthcare, authentication is used to verify the identity of healthcare professionals and care organizations. This is important to ensure that only authorized entities can access sensitive information and resources.
+
+The verified identity is often used downstream for authorization and accounting. Authorization is the process of granting access to resources based on the identity of an entity. Accounting is the process of tracking the actions of an entity.
+
+#### Requirements
+
+There exists a multitude of authentication solutions, therefore it is important to define the requirements for authentication in the context of healthcare. The requirements are:
+
+- Work with identity claims from the authoritative sources
+- Support combinations of identity claims from different trusted issuers
+- Support for use-cases with and without an end-user (healthcare professional)
+- Support for authorizing vendors to act on behalf of a (care) organization
+- Support for limiting the scope of a vendor's authorization
+- Support for authorizing (care) organizations to act on behalf of a healthcare professional
+- User-friendly solution for healthcare professionals
+- Cost-efficient solution for care organizations
+- Flexible solution that can be adapted to different use-cases
+- Peer-2-peer trust which allows direct interactions between the involved parties without involvement of a third party
+- No single point of failure
```

Comment:
when using VCs, you still might (have to) check revocation, meaning the issuer could still be a single point of failure, depending how you treat failed revocation checks